### PR TITLE
email verification in REGISTER/CONFIRMAGREEMENT

### PR DESCRIPTION
**Pull request by silentwings**
can be tested on the dev lobbyserver 78.46.100.157:7200

### CODE REVIEW

File: LuaMenu/widgets/chobby/components/login_window.lua [hunk 1]
```diff
@@ -158,10 +158,43 @@ function LoginWindow:init(failFunction, cancelText, windowClassname)
...
+	else
+		self.txtEmail = Control:New {}	
+		self.ebEmail = Control:New {}
+	end
+		
 	self.cbAutoLogin = Checkbox:New {
 		x = 15,
 		width = 215,
-		y = 210,
+		y = 250,
```
Comment (gajop) [inline LuaMenu/widgets/chobby/components/login_window.lua]: What about all these numbers?

File: LuaMenu/widgets/chobby/components/login_window.lua [hunk 2]
```diff
@@ -176,16 +209,16 @@ function LoginWindow:init(failFunction, cancelText, windowClassname)
 	self.txtError = TextBox:New {
 		x = 15,
 		right = 15,
-		y = 174,
+		y = 214,
 		height = 90,
 		text = "",
-		fontsize = Configuration:GetFont(3).size,
+		fontsize = Configuration:GetFont(2).size,
```
Comment (gajop) [inline LuaMenu/widgets/chobby/components/login_window.lua]: Is this wanted?

File: LuaMenu/widgets/chobby/components/login_window.lua [hunk 3]
```diff
@@ -228,7 +261,7 @@ function LoginWindow:init(failFunction, cancelText, windowClassname)
 	}
 	
 	local ww, wh = Spring.GetWindowGeometry()
-	local width, height = 430, 390
+	local width, height = 430, 430
```
Comment (gajop) [inline LuaMenu/widgets/chobby/components/login_window.lua]: This too (last I mention)

**Comment by silentwings** [comment]
The changes to positions are to allow an extra input box for email addresses.

I don't have any knowledge of ZKs protocol, or know where to find any documentation for it. Afaik this PR not affect the cmds sent unless protocol=="spring"

**Comment by silentwings** [comment]
I don't have any knowledge of ZKs protocol, or know where to find any documentation for it. Afaik this PR now does not affect the cmds sent unless protocol=="spring".

**Comment by gajop** [comment]
I mainly wanted to confirm that GUI still works well, with decent layout and behavior. I'll check it out when I have some time!

**Comment by gajop** [comment]
Thanks!

**Pull request merged**